app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,28 @@
 class PostsController < ApplicationController
-  def show; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts.order(created_at: :desc).paginate(page: params[:page], per_page: 2)
+  end
 
-  def index; end
+  def show
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+    @comment = Comment.new
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.create(author_id: params[:post][:user_id], title: params[:post][:title],
+                        text: params[:post][:text].gsub(/\r\n?/, '<br>'))
```

Comment:
[OPTIONAL]

- I think it will be better if you use strong params to handle your post-creation parameter. This way your app will follow the Sigle responsibility principle and also easier to read. 

e.g 
```
 private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```

I think this[ lesson](`https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.`) will help you learn about strong params 😄